tests/Docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM mcr.microsoft.com/powershell:latest
+ARG DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /tmp
+
+# Install base packages
+RUN apt-get update -qq
+RUN apt-get install wget vim git curl software-properties-common -y
+
+# Install neovim
+RUN add-apt-repository ppa:neovim-ppa/stable
+RUN apt-get update -qq
+RUN apt-get install neovim -y
+
+# Install nodejs
+RUN bash -c 'bash <(curl -sL install-node.now.sh/lts) --yes'
+
+# Create paswordless vim and nvim users
+RUN adduser --disabled-password --gecos "" vim
+RUN adduser --disabled-password --gecos "" nvim
+
+# Configure vim as vim user.
+USER vim
+WORKDIR /home/vim
+
+RUN curl -sfLo ~/.vim/autoload/plug.vim --create-dirs \
+    https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
+
+RUN echo "call plug#begin('~/.vim/plugged')\n\
+Plug 'neoclide/coc.nvim', {'branch': 'release'}\n\
+Plug 'bling/vim-airline'\n\
+Plug 'sheerun/vim-polyglot'\n\
+call plug#end()" >> ~/.vimrc
+
+RUN vim +PlugInstall +qall
+RUN vim -c 'CocInstall -sync coc-powershell|q'
+
+RUN rm -rf /tmp/coc-nvim*
+
+
+# Configure nvim as nvim user.
+USER nvim
+WORKDIR /home/nvim
+
+RUN curl -fLo ~/.local/share/nvim/site/autoload/plug.vim --create-dirs \
+    https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
+
+RUN mkdir -p ~/.config/nvim
+
+RUN echo "call plug#begin('~/.local/share/nvim/plugged')\n\
+Plug 'neoclide/coc.nvim', {'branch': 'release'}\n\
+Plug 'bling/vim-airline'\n\
+Plug 'sheerun/vim-polyglot'\n\
+call plug#end()" >> ~/.config/nvim/init.vim
+
+RUN nvim +PlugInstall +qall
+RUN nvim -c 'CocInstall -sync coc-powershell|q'
+
+USER root
+WORKDIR /tmp
+
+CMD ["/bin/bash"]
```

Comment:
Perhaps we should change this to `pwsh` since... It's a PowerShell container 😁

Reply:
How does it look now ?
Amended with both suggestions.